drivers/wifi/uwp/wifi_cmdevt.h

```diff
@@ -8,9 +8,9 @@
 #define __WIFI_CMDEVT_H__
 
 #define MAX_SSID_LEN (33) /* SSID end with 0 */
-#define MAX_KEY_LEN (128)
+#define MAX_KEY_LEN (128) /* FIXME: Max size 64 */
+#define MAX_AP_KEY_LEN (64)
 #define ETH_ALEN (6)
-#define INI_SIZE (1600)
 #define IPV4_LEN (4)
```

Comment:
This is already defined in wifi_main.h, can you remove it.

Reply:
Removed from wifi_main.h